Tools/clinic/clinic.py

```diff
@@ -4245,8 +4237,7 @@ def directive_version(self, required):
             fail("Insufficient Clinic version!\n  Version: " + version + "\n  Required: " + required)
 
     def directive_module(self, name):
-        fields = name.split('.')
-        new = fields.pop()
+        fields = name.split('.')[1:]
```

Comment:
pop() removes the last item in the list, no?

```suggestion
        fields = name.split('.')[:-1]
```

Reply:
Good catch. Yes, `[1:]` is definitely wrong. Strangely, `make clinic` does not complain or generate incorrect.